db/data.py

```diff
@@ -460,3 +461,32 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def check_low_level_duplicates(data):
+    """
+        Check if data comes from an audio file already analysed
+        Uses the md5_encoded field
+    Args: data
+        A dictionary containing the low-level data extracted using essentia
+    """
+    if 'md5_encoded' in data['metadata']['audio_properties']:
```

Comment:
If we check for this field in `SANITY_CHECK_KEYS`, perhaps we don't need to have a separate check here